README.md

```diff
@@ -49,4 +49,4 @@ To contribute to the tutorials, please check out our [Contributing Guidelines](.
 | [Serializing Pipelines](./tutorials/29_Serializing_Pipelines.ipynb)                | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/29_Serializing_Pipelines.ipynb) |
 | [Preprocessing Different File Types](./tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb)                | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb) |
 | [Metadata Filtering](./tutorials/31_Metadata_Filtering.ipynb)                | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/31_Metadata_Filtering.ipynb) |
-
+| [Language Classification in Documents and Queries](./tutorials/31_Metadata_Filtering.ipynb)                | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/32_Language_Classification_In_Documents_And_Queries.ipynb) |
```

Comment:
The 'in' in the title doesn't seem right to me. How about: 

'Classifying Document and Query Languages'